__tests__/app.test.js

```diff
@@ -264,4 +264,17 @@ describe("/api/articles/:article_id", () => {
             expect(body.message).toBe('Invalid Id Type')
         })
     })
+})
+
+describe("/api/comments/:comment_id", () => {
+    test("DELETE: 204 - responds with correct status code", () => {
+        return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(() => {
+            return request(app)
+            .get("/api/comments/1")
+            .expect(404)
+        })
+    })
```

Comment:
You don't need to follow the DELETE test with a GET request that expects a 404 error. This part can be removed 😄